dspace-api/src/main/java/org/dspace/app/audit/AuditEvent.java

```diff
@@ -0,0 +1,194 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.audit;
+
+import java.util.Date;
+import java.util.UUID;
+
+
+/**
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ * @author Stefano Maffei (stefano.maffei at 4science.com)
+ */
+public class AuditEvent {
+    private UUID uuid;
+    private UUID epersonUUID;
+    private UUID objectUUID;
+    private String objectType;
+    private UUID subjectUUID;
+    private String subjectType;
+    private String eventType;
+    private Date timeStamp;
+    private String detail;
+    private String metadataField;
+    private String value;
+    private String authority;
+    private Integer confidence;
+    private Integer place;
+    private String action;
+    private String checksum;
+
+    private final static String LOG_FIELD_SEPARATOR = " || ";
+
+    public UUID getUuid() {
```

Comment:
When writing some code, it's helpful if the IDE suggests and displays some information regarding the available class methods. Adding java docs to it, even we consider they are just gets and sets, is very helpful because sometimes we .